TheMovieDb/Modules/MainTab/MainTabRouter.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MainTabWireFrame.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 14/11/21.
+//  
+//
+
+import UIKit.UIImage
+
+class MainTabRouter: MainTabRouterProtocol {
+   
+    func buildControllers(build view: MainTabViewProtocol) {
+        
+        guard let view = view as? MainTabView else {return}
+        let homeView = HomeBuilder.createModule()
+        let home = view.templateNavigationController(
+            unselectedImage: UIImage(systemName: "house")!,
+            selectedImage: UIImage(systemName: "house.fill")!,
+            title: "Home",
+            rootViewController: homeView
+        )
+        
+        let searchView = SearchingBuilder.createModule()
+        let search = view.templateNavigationController(
+            unselectedImage: UIImage(systemName: "magnifyingglass")!,
+            selectedImage: UIImage(systemName: "text.magnifyingglass")!,
+            title: "Search",
```

Comment:
use a private constant enum to set names or use a Localizable

Reply:
fixed